### ISSUE TITLE
Low accuracy with CAER-S dataset

### ISSUE THREAD

**Issue by EHuba94** [zengqunzhao/EfficientFace#28]
Hello,
First thank you for sharing this.
I tried to run the code and my results are pretty bad.

```
Epoch: [0][  0/383]	Loss 0.2802 (0.2802)	Accuracy 12.500 (12.500)
Epoch: [0][ 40/383]	Loss 0.2424 (0.2451)	Accuracy 14.844 (15.377)
Epoch: [0][ 80/383]	Loss 0.2112 (0.2336)	Accuracy 18.750 (15.471)
Epoch: [0][120/383]	Loss 0.2191 (0.2280)	Accuracy 14.062 (15.341)
Epoch: [0][160/383]	Loss 0.2046 (0.2250)	Accuracy 13.281 (15.120)
Epoch: [0][200/383]	Loss 0.2290 (0.2229)	Accuracy 14.062 (14.976)
Epoch: [0][240/383]	Loss 0.2053 (0.2208)	Accuracy 14.062 (14.922)
Epoch: [0][280/383]	Loss 0.2143 (0.2190)	Accuracy 17.188 (14.822)
Epoch: [0][320/383]	Loss 0.2065 (0.2177)	Accuracy 15.625 (14.800)
Epoch: [0][360/383]	Loss 0.2144 (0.2163)	Accuracy 11.719 (14.699)
Test: [  0/164]	Loss 1.4293 (1.4293)	Accuracy 24.219 (24.219)
Test: [ 40/164]	Loss 2.4993 (1.7969)	Accuracy  3.125 (21.799)
Test: [ 80/164]	Loss 2.8212 (1.7931)	Accuracy  0.000 (30.498)
Test: [120/164]	Loss 4.1100 (2.4878)	Accuracy  0.000 (20.564)
Test: [160/164]	Loss 2.8931 (2.7365)	Accuracy  1.562 (15.703)
 * Accuracy 15.439
Current best accuracy: 15.439214706420898
```

I also looked at previous issues here and saw your recommendations about changing hyperparameters and restructuring the data set and I did all of it.
I also used the right Pre-trained models and LDG on the specific dataset. and still, those results are all the same.

Can you help me with some of your knowledge?
Regards

**Comment by EHuba94** [comment]
Did you use resnet50 for this dataset LDG?

**Comment by superoldzhang** [comment]
I have the same problem, have you solved it?
